test/distributed/org/apache/cassandra/distributed/upgrade/MixedModeBatchTestBase.java

```diff
@@ -62,6 +62,8 @@ protected void testSimpleStrategy(Semver from, Semver to, boolean isLogged) thro
         .setup(cluster -> {
             cluster.schemaChange("CREATE KEYSPACE test_simple WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 2};");
             cluster.schemaChange("CREATE TABLE test_simple.names (key int PRIMARY KEY, name text)");
+            if (isLogged)
```

Comment:
when logged we cause messages to get dropped which leads to timeouts.  These now make it to the cluster error handler which fails the build